DEPRECATION.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+Creation: 2023-02-24
```

Comment:
Which date format you're preferring?  Here it is `YYYY-MM-DD` but in the table and latest update it is DD-MM-YYYY or MM-YYYY. 

Please pick one for all document for consistency.

Reply:
sorry, my bad.
will be going with DD-MM-YYYY format.